scripts/conflicts/diff.js

```diff
@@ -22,35 +28,94 @@ console.log(`Generating diffs for conflicts on ${server}`);
 
 const DB = new PouchDB(server);
 
-DB.query('medic-conflicts/conflicts')
+const writeToFile = function(path, content) {
+  content  = JSON.stringify(content, null, 3); 
+  fs.writeFile(path, content, (err) => {
+    if(err){
+      console.log(err);
+    }
+  });
+};
+
+const createDirectory = function(conflictDirectoryPath) {
+  if (!fs.existsSync(conflictDirectoryPath)){
+    try {
+      fs.mkdirSync(conflictDirectoryPath);
+    } catch(err){
+      console.log(err);
```

Comment:
We should provide some context about where this error is being thrown. 
What would happen if you can;t create the folder (because of permissions for example)? Every next write is going to fail and the result will be a pagefull of hard to read errors. 
I think we should hard fail and stop execution if we can't create the folder or can't write a file. 

```suggestion
      console.error('Error when creating conflict folder', err);
      throw err;
```